h/traversal/__init__.py

```diff
@@ -76,4 +76,5 @@
     "UserByNameRoot",
     "UserByIDRoot",
     "UserRoot",
+    "GroupContext",
```

Comment:
Exporting `h.traversal.GroupContext` here so I can import it elsewhere for type annotations